buildlib/test.sh

```diff
@@ -0,0 +1,132 @@
+#!/bin/bash -eExl
+#
+# Testing script for SparkUCX
+#
+# Copyright (C) Mellanox Technologies Ltd. 2019.  ALL RIGHTS RESERVED.
```

Comment:
pls add a list with all env vars, if required or optional, and default values, in the header comment of the file
like in ucx/test_jenkins.sh